libdevcore/CommonIO.h

```diff
@@ -32,7 +32,10 @@ namespace dev
 
 /// Retrieve and returns the contents of the given file as a std::string.
 /// If the file doesn't exist or isn't readable, returns an empty container / bytes.
-std::string contentsString(std::string const& _file);
+std::string readFileAsString(std::string const& _file);
+
+/// Retrieve and returns the contents of standard input (until EOF).
+std::string readStandardInput();
```

Comment:
Alternatively instead of this we could have `readEntireStream(istream& _stream);`